### PR TITLE
chore: fix company name

### PR DESCRIPTION
**Pull request by mvayngrib**
depends on https://github.com/ExodusMovement/exodus-hydra/pull/6610

### CODE REVIEW

**Review by kewde — DISMISSED** [review]
utACK

**Review by sparten11740 — APPROVED** [review]

File: package.json
Comment (olistic) [inline package.json:1]: @mvayngrib unrelated change?

**Review by olistic — APPROVED** [review]
utACK

File: package.json
Comment (mvayngrib) [reply package.json:1]: that eslint plugin validates `author` in package.json

**Review by sparten11740 — APPROVED** [review]

**Review by diegomura — APPROVED** [review]
utACK
